confluent_kafka/src/confluent_kafka.c

```diff
@@ -1959,6 +1959,13 @@ PyObject *cfl_PyObject_lookup (const char *modulename, const char *typename) {
                 return NULL;
         }
 
+        if (!module) {
```

Comment:
Thanks for finding this, but I think the initial intent was for the if-statement above to handle this, could you change that one instead to check for module rather than modulename, and remove the first %s and modulename from the argument list.

Also, change it to PyExc_ImportError.

Reply:
Thanks for the review.  I've updated the code.
```
brian@dev-brian-1:~/code/confluent-kafka-python$ python ~/list.py
Traceback (most recent call last):
  File "/home/brian/list.py", line 5, in <module>
    print producer.list_topics(topic=topic).topics.values()
ImportError: Module not found when looking up confluent_kafka.admin.ClusterMetadata
```